common/config/babel/.babelrc.js

```diff
@@ -10,7 +10,7 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
       features: [
         // Flag to add API only available in beta calling SDK to mocks and internal types.
         // This feature should be stabilized whenever calling SDK is stabilized.
-        'calling-1.4.3-beta.1',
+        'calling-1.5.4-beta.1',
```

Comment:
It might be an outdate question, but @prprabhu-ms might help with answering this:
Why don't we call it 'calling-beta-sdk'? Are we intentionally need to modify all the comments each time when we update the sdk version?

Reply:
And all the api files need to be updated just because of those comments are included - which makes reviewing update beta much more difficult

Reply:
I could change it to that. I think at the time we wanted to be as specific as possible.